### DISCUSSION TITLE
HomePod als Klingel verwenden

### DISCUSSION THREAD

**Discussion by mbrockeu** [brutella/hkknx-public#191] [Ideas]
Ich hatte letztens eine Automation gesehen (https://youtu.be/T45WSUNHrx4?t=158) wo der HomePod als Klingel verwendet wurde. Dies ist ja seit iOS 14 möglich. Allerdings benötigt man dafür auch eine bestimmte Klingel.

Wäre es denkbar, dass man diese virtuelle Videoklingel auch hier implementieren könnte, so dass diese ggfs. auf eine GA hört und dann der HomePod entsprechend klingelt? 
So könnte man ja einfach über einen Taster einen Binäreingang ansteuern und diese Informationen weitergeben.

**Comment by chef76** [comment]
Das geht jetzt schon mit Bordmitteln. 

HomeKit Automation: Wenn Deine GA aktiv -> Kurzbefehl ausführen. In dem Kurzbefehl lässt Du den HomePod ein Klingel mp3 spielen. Fertig. Funktioniert ohne Probleme. Kannst du bei Andreas im nächsten Automationsvideo anschauen, wenn er es veröffentlicht, nur melde ich damit, wenn mein Trockner fertig ist.

**Reply by mbrockeu** [reply]
Ich weiß dass es geht, finde die Lösung aber nur gebastelt und so zuverlässig wie Apple Music auf dem HomePod funktioniert wäre dies eine alternative Lösung.

Zumal es ja dann quasi eine richtige Klingel ist. Auch wenn das Video nicht funktioniert.

**Reply by chef76** [reply]
Du willst also, dass sich ein KNX Taster als virtuelle Videoklingel verhält. Und das wäre dann nicht "gebastelt" 🤔

**Reply by mbrockeu** [reply]
Naja man könnte jetzt sagen, eine nicht zertifizierte Schnittstelle ist sowieso „gebastelt“. 

Ich möchte einen Klingeltaster einfach nur durchreichen und keine weitere Logik in HomeKit bauen mit den Automationen. 

Ich kann deinen Vorschlag gerne noch mal testen, aber die Reaktionszeit war schon langsam. Außerdem hätte der Kurzbefehl zur Folge das eine Musikwiedergabe dauerhaft verändert wird. Und wenn ich alle HomePods Ansteuere könnte das ggfs. Problematisch werden wenn er die Kopplung nicht aufhebt. 

Das die virtuelle Klingel auch gebastelt ist stelle ich nicht in Abrede. Eleganter wäre es aber schon. Oder nicht? Nur das man den Klingelton glaube nicht wählen kann. 😜

**Reply by chef76** [reply]
Dass das nicht ganz verzögerungsfrei geht ist klar, aber ansonsten habe ich da keine Probleme. War ja auch nur als Denkanstoß gemeint. Wir basteln hier alle nur jeder auf seine Weise ...

**Comment by mukku82** [comment]
Hi,
da ich auch gerade nach einer geeigneten Lösung bin und ich  nicht noch weitere Systeme erlenen möchte wäre ein Dummy welcher den Homepod klingeln lassen würde perfekt. 
Die Lösung über die Homekit Automation geht bei mir leider nicht da ich kein Music Abo habe.

@brutella Meinst du sowas wäre umsetzbar?

Grüße

**Reply by chef76** [reply]
Wie kommst du darauf, dass man dafür ein Apple Music Abo braucht?

**Reply by mukku82** [reply]
Weil ich an dem Punkt MP3 Datei auswählen er mir sagt das ich ohne Music Abo keine Datei auswählen kann. Er sagt mir: Zum Hinzufügen von Audio zu einer Automation benötigst du eine aktive Mitgliedschaft bei Apple Music.

**Reply by chef76** [reply]
Ok, das kann nicht verneinen, da ich ein Apple Music Abo habe. Wenn das aber die Vorraussetzung ist, wird die Brücke da auch nichts abspielen können. :-(. Ich spiele ein eigenes MP3 ab, das ich über iTunes integriert habe.
